com.unity.cinemachine/Tests/CmCameraProceduralBehaviourCacheTests.cs

```diff
@@ -82,7 +86,8 @@ public IEnumerator AddAndDestroyAllOneByOne()
                 
                 var component = m_CmCamera.gameObject.GetComponent(cmComponent);
                 RuntimeUtility.DestroyObject(component);
-                Assert.True(m_CmCamera.PipelineCacheInvalidated); // invalid pipeline after add
+                Assert.True(m_CmCamera.PipelineCacheInvalidated); // invalid pipeline after destroy
+                yield return null;
```

Comment:
Need to wait a frame here, otherwise the Destroyed component might not be destroyed yet.